db/primary_migrate/20220105143455_create_sp_return_logs_requested_at_date_issuer_index.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSpReturnLogsRequestedAtDateIssuerIndex < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    safety_assured { execute "CREATE INDEX CONCURRENTLY index_sp_return_logs_requested_at_date_issuer ON public.sp_return_logs USING btree ((requested_at::date), issuer) WHERE (returned_at IS NOT NULL);" }
```

Comment:
personally I'd put this on multiple lines so it's easier to read but not a requirement or anything

```suggestion
    safety_assured do
      execute <<-SQL
        CREATE INDEX CONCURRENTLY index_sp_return_logs_requested_at_date_issuer
        ON public.sp_return_logs
        USING btree ((requested_at::date), issuer)
        WHERE (returned_at IS NOT NULL);
      SQL
    end
```

Reply:
Thanks, this is much better